deployments/kubernetes/install/helm/istio-cni/templates/istio-cni.yaml

```diff
@@ -164,3 +191,33 @@ subjects:
   name: istio-cni
   namespace: {{ .Release.Namespace }}
 {{- end }}
+    {{- if .Values.repair.enabled }}
```

Comment:
nit: indent

Reply:
how come we have the conditional here but either way we create the container?

Reply:
Oversight, will fix it.